net.sourceforge.GTKWave.yml

```diff
@@ -0,0 +1,49 @@
+app-id: net.sourceforge.GTKWave
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+command: gtkwave
+rename-icon: gtkwave
+rename-desktop-file: gtkwave.desktop
+rename-appdata-file: gtkwave.appdata.xml
```

Comment:
if you have it upstream (as the rename seem to indicate) there is no reason to have a copy here.

Reply:
I tried with that initially, but it looks like there are some necessary tags missing that are required for flatpak. So, I've created a copy of it locally.

Reply:
usually what is done is patches. So they get submitted upstream too.